presto-hive/src/main/java/io/prestosql/plugin/hive/security/SqlStandardAccessControl.java

```diff
@@ -264,7 +264,7 @@ public void checkCanRenameColumn(ConnectorSecurityContext context, SchemaTableNa
     }
 
     @Override
-    public void checkCanSelectFromColumns(ConnectorSecurityContext context, SchemaTableName tableName, Set<String> columnNames)
+    public void checkCanSelectFromColumnsWithMetadata(ConnectorSecurityContext context, SchemaTableName tableName, Set<ColumnMetadata> columns)
```

Comment:
I think we should sacrifice backward compatibility and change the parameter type without changing method name (adding new method)

@findepi @electrum what do you think?